gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/store/GemFireStore.java

```diff
@@ -1473,7 +1473,11 @@ private void bootRegions() throws StandardException {
               dsf.setMaxOplogSize(DiskStoreFactory.DEFAULT_MAX_OPLOG_SIZE);
             }
             else {
-              dsf.setMaxOplogSize(10);
+              if (this.myKind.isAccessor()) {
+                dsf.setMaxOplogSize(1);
+              } else {
+                dsf.setMaxOplogSize(10);
+              }
```

Comment:
Only _BACKUPGFXD-DEFAULT-DISKSTORE.if_ and _DRLK_IFGFXD-DEFAULT-DISKSTORE.lk_ are created in lead directory. No crf and drf files generated.

Size of _BACKUPGFXD-DEFAULT-DISKSTORE.if_ is **1.0 MB**.
Size of _DRLK_IFGFXD-DEFAULT-DISKSTORE.lk_ is **0 bytes**